addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -60,10 +60,15 @@ $o-btn-outline-border-width-defaults: () !default;
 }
 
 // Highlight '.btn-group's active buttons
-.btn-group .btn-light.active {
-    box-shadow: inset 0 0 0 $border-width $component-active-bg;
-    border-color: transparent;
-    background-color: mix($component-active-bg, $light, 10%);
+.btn-group {
+    & .btn-check:checked + label.btn-light {
+        border-color: $component-active-bg;
```

Comment:
Removing the box-shadow from the active button seems fine on light background, however, on a dark surface, I feel like the active button looks smaller than the not active one with this change, wdyt ?

| This PR | Master |
|--------|--------|
| <img width="125" alt="image" src="https://github.com/odoo/odoo/assets/128030743/7e927e54-b93b-4d54-a691-ca817d84c109"> | <img width="121" alt="image" src="https://github.com/odoo/odoo/assets/128030743/a1347d58-d8d1-49c4-8cd2-db13ba95b15f"> |